cea/datamanagement/data_helper.py

```diff
@@ -212,6 +212,17 @@ def calc_mainuse(uses_df, uses):
     :rtype mainuse: ndarray
 
     """
+
+    # print a warning if there are equal shares of more than one "main" use
+    indexed_df = uses_df.set_index('Name')
+    for building in indexed_df.index:
+        mainuses = [use for use in uses if
+                    (indexed_df.loc[building, use] == indexed_df.max(axis=1)[building]) and (use != 'PARKING')]
+        if len(mainuses) > 1:
+            print '%s has equal share of %s; the construction properties and systems for %s will be used.' % (
+            building, ' and '.join(mainuses), mainuses[0])
```

Comment:
Where is mainuses[0] used later on in the code?  I guess that is the selected use and will tell which building properties to use?

Reply:
No, not really. The main use is actually calculated in a non-iterative way in the next section, but I was not able to get two or more "main" uses using that method. So `mainuses[0]` is only used to print the requested message. Of course the code can be adapted so that there is only one `mainuse` definition.